### ISSUE TITLE
create path clicking on arrow on flag, create flag on existing path using doubleclick  ...

### ISSUE THREAD

**Issue by p1p1p1redeye** [freeserf/freeserf#470]
it shoud be nice if player can start creating path only by double clicking on arrow at bottom of flag,

it shoud be nice if player can add flag on existing path clicking on some poing where it is possible to add flag (yellow flag is displayed) and add it there by doubleclicking there!!

**Comment by p1plp1** [comment]
i have it allready done, now im fixing game bugs ... can not commint changes, cmake is not working correctly for me.

**Comment by tlongstretch** [comment]
did you try the guide I posted a link to in the other issue?  https://github.com/freeserf/freeserf/wiki/Building-from-Source-on-Windows

**Comment by p1plp1** [comment]
yes, it works maybe with visual studio, sources are VS specific, I allready modified problematic parts for mingw compiler, cmake was problematic, it didnt compile runnable exe file, so i modified libs order to do it properly, whatever, ill fix some more bugs in my free time and when it runs stable i'll look at that problem or ill send u current code, exe, compiller and environment settings, then u can try it under VS if it works without problems, now i have only little bit of free time, it shoud be better around xmas - maybe i finish fixing bugs and then ill look on that crazy stats panels :) there is allmost nothing working properly.

**Comment by tlongstretch** [comment]
I don't actually have commit rights, but if you send me your code I'll try it on my current codebase which has AI players.  Once I am done fixing my code I will submit it

**Comment by p1plp1** [comment]
ok it is still buggy, some things are fixed, mostly game crashing problems, but not yet all im thinking how to do serf delete just independent on serfs update, thats currently most problematic part ...
